### PR TITLE
Add live arg to list()

### PR DESCRIPTION
**Pull request by Powersource**
Fixes https://github.com/ssbc/ssb-tribes2/issues/23

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -44,7 +44,7 @@
     "pull-paramap": "^1.2.2",
     "pull-stream": "^3.7.0",
     "ssb-bfe": "^3.7.0",
-    "ssb-box2": "^4.0.0",
+    "ssb-box2": "github:ssbc/ssb-box2#list-live",
```
Comment (Powersource) [inline package.json:47]: Needs merge and major release https://github.com/ssbc/ssb-box2/pull/25

File: test/list-and-get.test.js [hunk 2]
```diff
@@ -139,3 +141,71 @@ test('list', (t) => {
...
+    .addMembers(group.id, [bobRoot.id])
+    .catch((err) => t.fail(err))
+  t.pass('bob was added')
+
+  await replicate(alice, bob).catch(t.fail)
+  console.log('finished replicating')
+  t.pass('alice and bob replicated the invite')
+
+  console.log('about to accept', group)
+  const acceptedInvite = await bob.tribes2.acceptInvite(group.id).catch(t.fail)
```
Comment (Powersource) [inline test/list-and-get.test.js:202]: Test breaks somewhere around here, it logs e.g.

```
accepted invite undefined
```

File: package.json (hunk 1 above)
Comment (staltz) [reply package.json:46]: Can you merge that and update this?

**Comment by Powersource** [comment]
@staltz box2 5 released and used

File: index.js [hunk 3]
```diff
@@ -342,38 +337,50 @@ module.exports = {
...
-                  }
-                })
-              )
-
-              return cb(null, source)
-            })
+            pull(
+              ssb.box2.listGroupIds(),
+              pull.collect((err, groupIds) => {
+                if (err)
```
Comment (staltz) [inline index.js:343]: Could you bring back the `prettier-ignore` for this line? It's a pattern we use in ssb-db2 too, to consistently have the error line not take up that much space.

**Review by staltz — CHANGES_REQUESTED** [review]
Just one tiny thing to revert, then this is good to merge. Good job

File: index.js (hunk 3 above)
Comment (Powersource) [reply index.js:343]: i think this got lost in a merge. i'll fix

**Comment by Powersource** [comment]
released in 0.4.0

**Pull request merged**